scripts/superdesk/editor/spellcheck/spellcheck.spec.js

```diff
@@ -73,6 +74,22 @@ describe('spellcheck', function() {
         expect(errors.length).toBe(0);
     }));
 
+    it('can report error if paragraph starts with small letter', inject(function(spellcheck, api, $rootScope) {
+        var p = createParagraph('test');
+
+        spyOn(api, 'save');
+        spellcheck.errors(p).then(assignErrors);
+        $rootScope.$digest();
+        expect(errors.length).toBe(1);
+
+        spellcheck.addWordToUserDictionary('test');
```

Comment:
not sure this test works, because it's mocking dictionaries, so would be better to test with some word that is already there